grant_access/gpu-runner-pr/magma.txt

```diff
@@ -0,0 +1 @@
+libmagma
```

Comment:
Just checking. Do you want to build PRs too?

Reply:
Yes, please. Why not?